src/Compilers/CSharp/Test/Symbol/Symbols/Metadata/MetadataTypeTests.cs

```diff
@@ -140,12 +140,12 @@ public void MetadataTypeSymbolGenClass02()
             Assert.False(type1.IsVirtual);
             Assert.False(type1.IsOverride);
 
-            // 4 nested types, 64 members overall
-            Assert.Equal(64, type1.GetMembers().Length);
-            Assert.Equal(4, type1.GetTypeMembers().Length);
+            // 4 nested types, 67 members overall
+            Assert.Equal(67, type1.GetMembers().Length);
+            Assert.Equal(3, type1.GetTypeMembers().Length);
             // IDictionary<TKey, TValue>, ICollection<KeyValuePair<TKey, TValue>>, IEnumerable<KeyValuePair<TKey, TValue>>, 
             // IDictionary, ICollection, IEnumerable, ISerializable, IDeserializationCallback
-            Assert.Equal(8, type1.Interfaces().Length);
+            Assert.Equal(10, type1.Interfaces().Length);
 
             var fullName = "System.Collections.Generic.Dictionary<TKey, TValue>";
```

Comment:
This are just differences between the frankenstein mscorlib we were using and the correct reference assembly.